README.md

```diff
@@ -22,21 +22,48 @@ of CI workflows for release automation.
 
 ## Setup
 
-If not already set up, install west and other required tools. Follow the steps
-[here](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html)
-from
-[Install the required tools](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#install-the-required-tools)
-up to (including)
-[Install west](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#install-the-required-tools).
+If not already set up, install west and other required tools.
+Follow the steps [here](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html)
+from [Install the required tools](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#install-the-required-tools)
+up to (including) [Install west](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#install-west).
+
+Also install [east](https://github.com/IRNAS/irnas-east-software).
 
 Then follow these steps:
 
 ```bash
 west init -m https://github.com/IRNAS/<repo-name> <repo-name>
-cd <repo-name>/
+cd <repo-name>/project
 west update
-# remember to source zephyr env
-source zephyr/zephyr-env.sh
+
+# set up east globally (this only needs to be done once on each machine)
+east sys-setup
+# install toolchain for the version of NCS used in this project
+east update toolchain
+```
+
+## Building and flashing
+
+To build the application firmware:
+
+```bash
+cd app
+east build -b <board_name> -u <build_type>
```

Comment:
Add a step in the checklist which says to update the values in this command with correct ones.
(what is the board name, is there a build type, etc.)

Now that I am thinking about this, this is something that can really be done before first release.